frontend/tsconfig.json

```diff
@@ -15,7 +15,7 @@
         "noImplicitAny": true,
         "strict": true,
         "strictBindCallApply": true,
-        "strictFunctionTypes": true,
+        "strictFunctionTypes": false,
```

Comment:
@jeswanke What is your reason for changing these settings?

Reply:
(both `strictFunctionTypes` and `target`)

Reply:
...patternfly's demo has strict turned off--they're willing to fix, but it won't be time for 2.7--alternatively we can disable whatever ts errors occur

...the patternfly layout requires us to extend the BaseLayout and override its methods--which isn't supported in es5 but is supported in es2015

all in all, using patternfly topology requires a closer integration then table or button